### DISCUSSION TITLE
Frontend implementation configurable via env vars

### DISCUSSION THREAD

**Discussion by adriangb** [realworld-apps/realworld#879] [Q&A]
I'm developing a backend and would like to deploy it with a frontend just as a demo. Because I'm not working on the frontend, I'd like to keep that part as simple as possible. Are there any frontends that have a published docker image configurable only with env vars, so that I can point it at my backend and call it a day?

**Comment by geromegrignon** [comment]
Hi @adriangb, we don't have a given list for such a request.
However, I would discourage doing so: By providing a public demo, we faced some unwanted content in the past we couldn't moderate easily. We had to create a brand new server to hide users' content from the public.

**Reply by adriangb** [reply]
Yeah I did see the issues w.r.t. content moderation.
I still think that having a front end that can be spun up easily as a container makes sense. There's a lot of use cases outside of a long lived public (and publicized) demo:

1. Local development 
2. Short lived demos, e.g. for a conference 
3. Tutorials

**Reply by adriangb** [reply]
And just to clarify: this isn't a _request_, I intended it to be a question along the lines of _does anyone know of a..._. Apologies if it sounded like a request 😄

**Reply by geromegrignon** [reply]
It didn't felt like a request for me :)
I agree about use cases

**Reply by adriangb** [reply]
I think it shouldn't be too hard to take one of the existing frontends and make this change. I can do it myself (fork a frontend repo, upload to my personal dockerhub), but then I don't think it would be very useful because no one will discover it. If you feel good about the use cases, would you be open to doing this via official channels? I'm happy to send a PR.
